source/backend/api/Areas/Property/Models/Search/PropertyFilterModel.cs

```diff
@@ -49,10 +55,32 @@ public PropertyFilterModel(Dictionary<string, Microsoft.Extensions.Primitives.St
             var filter = new Dictionary<string, Microsoft.Extensions.Primitives.StringValues>(query, StringComparer.OrdinalIgnoreCase);
 
             this.Sort = filter.GetStringArrayValue(nameof(this.Sort));
+            var tempSort = this.Sort.ToList();
+
+            // Convert sort to db format
+            for (int i = 0; i < this.Sort.Length; i++)
+            {
+                if (tempSort[i].StartsWith("Location"))
+                {
+                    tempSort[i] = tempSort[i].Replace("Location", "Address.MunicipalityName");
```

Comment:
I can't check the story, but are we sure location maps to municipality (I know  we also have a general location field in the db)

Reply:
yes, that is mapped on the frontend columns.

<img width="361" alt="image" src="https://github.com/bcgov/PSP/assets/12201292/d574a147-faf9-41fb-87d3-dd26245b7df1">